lib/3scale_toolbox/commands/application_command/apply_command.rb

```diff
@@ -186,6 +204,15 @@ def application_ref
           def option_redirect_url
             options[:'redirect-url']
           end
+
+          def printer
```

Comment:
Can we follow the same type of implementation that has been done in the printer method of lib/3scale_toolbox/commands/activedocs_command/list_command.rb (instead of if/else a options.fetch with a default fallback)?
If so, can we apply that type of implementation to all commands?

Reply:
:+1: